Terminal.Gui/Drawing/ColorScheme.Colors.cs

```diff
@@ -173,4 +173,123 @@ public ColorScheme? this [string key]
 
         return ColorSchemes;
     }
+
```

Comment:
I don't think we should be putting active UI in a low-level class like `ColorScheme`.

This should go in `./Views/ColorPicker/ColorPicker.Prompt.cs` (and the other `./Views/ColorPicker*.cs` files should move to `./Views/ColorPicker`. 

I'd make `ColorPicker.Prompt` a `partial` of `ColorPicker`. Name the method `Prompt`.

In the future we can add other variants of `Prompt` (e.g. one that just does a single color).

Make sense?

Reply:
Done in https://github.com/gui-cs/Terminal.Gui/pull/3772/commits/52435e4943ccc615e115a311a9bb607681c4dfee.